Projects/Team2/derback top secret/Program №2.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+
+namespace ConsoleAppOOP1
+{
+    class Program
+    {
+        public class Person
+        {
+            int count;
+            string sexinfo;
+            public string name = "Vetal";
+            public int age = 18;
+            public int sex = 1;
+            public long telephonenumber = 0968484332;
+
+            public void PersonPrint()
+            {
+                switch (sex)
+                {
+                    case 1:
+                        sexinfo = "Man";
+                        break;
+                    case 2:
+                        sexinfo = "Woman";
+                        break;
+                    default:
+                        sexinfo = "Sex unassigned";
+                        break;
+                }
+                Console.WriteLine($"Name: {name}\nAge: {age}\nSex: {sexinfo}\nTelephone: {telephonenumber}");
+                Console.ReadKey();
+            }
+            public string Name()
```

Comment:
It is better to use the name SetName(), SetAge() etc. We need to understand just from a name what is the method for.